### PR TITLE
chore: use github packages

### PR DESCRIPTION
**Pull request by septs**

### CODE REVIEW

File: .npmrc [hunk 1]
```diff
@@ -0,0 +1,2 @@
+@dimensiondev:registry=https://npm.pkg.github.com/
+//npm.pkg.github.com/DimensionDev:_authToken=1bbc5896d56bf4135a1a9f9fd0ecfbcbb70a0977
```
Comment (Jack-Works) [inline .npmrc:2]: What is this token used for? Is it safe to commit to the git?

Comment (septs) [reply .npmrc:2]: https://github.com/orgs/DimensionDev/teams/dev/discussions/6

Comment (septs) [reply .npmrc:2]: safe

File: src/extension/background-script/SteganographyService.ts [hunk 2]
```diff
@@ -1,8 +1,8 @@
-import { encode, decode } from '@dimensiondev/stego-js/es/dom'
-import { GrayscaleAlgorithm } from '@dimensiondev/stego-js/es/grayscale'
-import { TransformAlgorithm } from '@dimensiondev/stego-js/es/transform'
+import { encode, decode } from '@dimensiondev/stego-js/dist-cjs/dom'
```
Comment (Jack-Works) [inline src/extension/background-script/SteganographyService.ts:1]: Why

Comment (septs) [reply src/extension/background-script/SteganographyService.ts:1]: https://github.com/DimensionDev/Stego-JS/pull/6

File: package.json [hunk 3]
```diff
@@ -45,8 +43,9 @@
     ]
   },
   "dependencies": {
-    "@dimensiondev/stego-js": "^0.11.0",
-    "@holoflows/kit": "https://github.com/DimensionDev/holoflows-kit",
+    "@dimensiondev/matrix-js-sdk-type": "unstable",
```
Comment (Jack-Works) [inline package.json:46]: Does this repo up to date with the upstream? (https://github.com/Jack-Works/matrix-js-sdk-type)

Comment (septs) [reply package.json:46]: synced, https://github.com/DimensionDev/matrix-js-sdk-type

**Comment by septs** [comment]
https://github.com/yarnpkg/yarn/issues/4451

File: src/social-network-provider/facebook.com/tasks/uploadToPostBox.ts [hunk 4]
```diff
@@ -3,7 +3,7 @@ import { untilDocumentReady } from '../../../utils/dom'
 import { getUrl, downloadUrl, pasteImageToActiveElements } from '../../../utils/utils'
 import Services from '../../../extension/service'
 import { decodeArrayBuffer } from '../../../utils/type-transform/String-ArrayBuffer'
-import { GrayscaleAlgorithm } from '@dimensiondev/stego-js/es/grayscale'
+import { GrayscaleAlgorithm } from '@dimensiondev/stego-js/cjs/grayscale'
```
Comment (Jack-Works) [inline src/social-network-provider/facebook.com/tasks/uploadToPostBox.ts:6]: What about these changes? Use both es and cjs version are not cool

Comment (septs) [reply src/social-network-provider/facebook.com/tasks/uploadToPostBox.ts:6]:
```
@dimensiondev/stego-js/esm
```

compiler module is `es6`

Comment (septs) [reply src/social-network-provider/facebook.com/tasks/uploadToPostBox.ts:6]: orignal `es` directory, output is `commonjs`
current `cjs` directory, output `commonjs`, `esm` directory output `esmodule`

**Pull request merged**